opencti-platform/opencti-front/src/schema/relay.schema.graphql

```diff
@@ -542,11 +542,13 @@ type StreamCollection {
   description: String
   filters: String
   groups: [Group]
+  live: Boolean
+  public: Boolean
 }
 
 type StreamCollectionConnection {
   pageInfo: PageInfo!
-  edges: [StreamCollectionEdge]
+  edges: [StreamCollectionEdge]!
```

Comment:
Inside element must always be set as mandatory now. Best pratice i think

Reply:
Done